CONTRIBUTING.md

```diff
@@ -55,37 +55,37 @@ Thank you! We'll try to get back to you as soon as possible.
 1. Install Rust using [rustup], which allows you to easily switch between Rust
    versions. Diesel currently supports Rust Stable, Nightly, Rust Beta.
 
-   If you want to run Diesel's test suite with _all_ supported features (extra
+   To run Diesel's test suite with _all_ supported features (extra
    lints and compiletest), you should use the same nightly version as Diesel's
-   continuous integration. You can find it by looking for a line like
+   continuous integration. Find it by looking for a line like
    `rust: nightly-2017-06-06` in the `.travis.yml` file. You can install and
    set a custom nightly version for a project using
    `rustup override add nightly-2017-06-06`.
 
 2. Install the system libraries needed to interface with the database systems
    you wish to use.
 
-   These are the same as when compiling diesel. In general, it is a good idea
-   to have _all_ drivers installed so you can run all tests locally.
+   These are the same as when compiling Diesel. It's generally a good idea
+   to install _all_ drivers so you can run all tests locally.
 
-   *Shortcut:* On macOS, you don't need to install anything to work with SQLite
-   and for PostgreSQL you'll only need the server (`libpq` is installed by
-   default). So, to get started, `brew install postgresql mysql` and follow the
+   *Shortcut:* On MacOS, you don't need to install anything to work with SQLite.
+   For PostgreSQL, you'll only need the server (`libpq` is installed by
+   default). To get started, `brew install postgresql mysql` and follow the
    instructions shown to set up the database servers.
 3. Clone this repository and open it in your favorite editor.
 4. Create a `.env` file in this directory, and add the connection details for
    your databases.
 
-   See [.env.sample](.env.sample) for an example that should work with a trivial
+   See [.env.sample](.env.sample) for an example that works with a trivial
    local setup.
 
    *Note:* If you didn't specify the MySQL user to be one with elevated
-   permissions, you'll want to a command like ```mysql -c "GRANT ALL ON
+   permissions, you'll want to make a command like ```mysql -c "GRANT ALL ON
```

Comment:
Oh, totally missed the verb here, thanks! I think it should be 'run', though.

Reply:
+1